orchestra/bots/staffbot.py

```diff
@@ -296,7 +296,7 @@ def get_staffing_request_metadata(self, staffing_request_inquiry):
             'workflow_description': workflow_description,
             'project_description': project_description,
             'detailed_description': detailed_description,
-            'task': staffbot_request.task
+            'available_datetime': staffbot_request.created_at
```

Comment:
does this affect anything other than the Available tasks table?

Reply:
gets used to template slack/email messages (they don't reference `task`) and the available tasks table